permissions-policy-explainer.md

```diff
@@ -337,9 +333,9 @@ The biggest visible change is that the `Feature-Policy` header is now spelled
 `Permissions-Policy`, and its value is a [Structured Field Value](https://httpwg.org/http-extensions/draft-ietf-httpbis-header-structure.html).
 (Technically, it's a structured Dictionary, whose names are the feature names,
 and values are either items or inner lists, but see the spec for all of the
-details there)
+details there.)
```

Comment:
Not outside of the parentheses? :)

Reply:
No, not when it's a full/standalone sentence, e.g.: https://www.thepunctuationguide.com/parentheses.html